src/goto-programs/class_identifier.h

```diff
@@ -15,16 +15,10 @@ Author: Chris Smowton, chris.smowton@diffblue.com
 class exprt;
 class namespacet;
 class symbol_typet;
-class struct_exprt;
 
 exprt get_class_identifier_field(
   const exprt &this_expr,
   const symbol_typet &suggested_type,
   const namespacet &ns);
 
-void set_class_identifier(
-  struct_exprt &expr,
-  const namespacet &ns,
-  const symbol_typet &class_type);
-
```

Comment:
While I appreciate the clean revert: isn't this actually a desirable place for `set_class_identifier` to be declared? (And, in line with that, the implementation in the .cpp file?)